libpod/networking_linux.go

```diff
@@ -36,7 +36,7 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-const (
+var (
```

Comment:
Can you keep this block `const` and then add a separate `var` block below for the slirp4netns bit?

Reply:
I have changed this I just don't want to kick off another CICD run until I get the other things sorted out.